webfriends-selenium-elements/src/main/java/com/github/bordertech/webfriends/selenium/smart/driver/SmartDriver.java

```diff
@@ -88,6 +143,32 @@ public <T extends SElement> T findWithText(final TagTypeSelenium<T> elementTag,
 		return findWithText(elementTag, text, true);
 	}
 
+
+	/**
+	 * Find the first element with the tag type and matching text.
+	 *
+	 * @param <T>        the element tag type
+	 * @param elementTag the element tag
+	 * @param text       the text of the button
+	 * @return the labeled element or null
+	 */
+	public <T extends SElement> T findWithText(final TagTypeSelenium<T> elementTag, final SearchContext searchContext, final String text) {
+		return getHelper().findWithText(this, searchContext, elementTag, text, true);
+	}
+
+	/**
+	 * Find the first element with the tag type and matching text.

```

Comment:
Javadoc comments on the findWithText methods seem to have the same text and params dont match the method. It would also be good to describe what each method does a bit differently.